Packs/CyberChef/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### CyberChef
+
+- Updated the Docker image to: *demisto/cyberchef:1.0.0.104247*.
+- Add cyberchef node package as dependency to execute cyberchef local
```

Comment:
```suggestion
- Added support for cyberchef node package as dependency to execute cyberchef locally.
```
See https://xsoar.pan.dev/docs/documentation/release-notes for more information.